src/teleport.cpp

```diff
@@ -70,19 +70,24 @@ void Teleport::addThing(Thing* thing)
 	return addThing(0, thing);
 }
 
-bool Teleport::checkInfinityLoop(Tile* destTile)
+bool Teleport::isHuge(const Tile* destTile)
 {
-	if (!destTile) {
-		return false;
-	}
```

Comment:
Leave the pointer check in, since `g_game.map.getTile` can return a nullptr and will crash the server

Reply:
The function was not designed to be passed a nullptr, since this verification is done before in line 92

Reply:
True, I looked at the diff in split view and looked at the code on the wrong side. The old code needed it because it was recursing with `return checkInfinityLoop(g_game.map.getTile(nextDestPos));`